+siibra/+core/Parcellation.m

```diff
@@ -35,53 +35,45 @@
             regions = webread(parcellation_json.links.regions.href);
             
             % store graph
-            parcellation.Graph = Parcellation.createParcellationTree(parcellation, regions);
-            
+            parcellation.regionTree = siibra.core.Parcellation.createParcellationTree(parcellation, regions);
```

Comment:
Another style for doing this is to use a [property access method](https://www.mathworks.com/help/matlab/matlab_oop/property-access-methods.html#:~:text=Property%20access%20methods%20execute%20specific,range%20restrictions%20(Validate%20Property%20Values) for the `RegionTree` property and to use a [persistent variable](https://www.mathworks.com/help/matlab/ref/persistent.html?searchHighlight=persistent%20variable&s_tid=srchtitle_persistent%20variable_1] within the method to store the value determined on the first access.  This would streamline constructor and avoid needing a hidden static method.

Reply:
I see. I like the fact that all atlases, parcellations, regions and spaces are initialized in the beginning. Your proposed approach would lazy initialize the regions on the first access. If you do not have a strong opinion on that I would like to keep it simple leave it as is.